### DISCUSSION TITLE
Global settings screen on first run

### DISCUSSION THREAD

**Discussion by NyarlathotepZERO** [sandboxie-plus/Sandboxie#2088] [Various ideas and suggestions]
It occurred to me that It's actually a little confusing on the first run of sandboxie when you are prompted with the "sandboxie has detected the following software applications in your system, click OK to apply configuration settings..." screen.

The items that show in this window each have their own check boxes but the screen prompt does not tell you if you need to use these check boxes for each of the applications or not. 

If I just click on OK as the window suggests (leaving all boxes unchecked) are these settings applied for each of them? This screen should probably be altered to make it a little clearer as to what it is doing and what the user needs to do.

**Comment by DavidXanatos** [comment]
yes you only need to click on ok

**Comment by isaak654** [comment]
@DavidXanatos @NyarlathotepZERO 
What about:
`The following software applications were detected in your system. It is recommended to review them by manually checking/unchecking the checkboxes and clicking OK to apply the changes. These configuration settings will have effect in all existing sandboxes and in any new sandboxes.` ?

**Comment by NyarlathotepZERO** [comment]
As David mentioned, it seems that nothing on this screen needs to be checked. After installation the appropriate templates have been applied automatically without any interaction whatsoever (even though the boxes were NOT checked). I still definitely think the process is slightly confusing, Isaak has the right idea. Some more appropriate text with a little less ambiguity would be a much more solid method to guide the installer. Either make the person installing, manually check each of the boxes; explaining that only checked boxes will be applied, or have all the boxes checked by default, explaining that the user can uncheck any/all boxes (at their own peril).
